src/lib/core/tests/BUILD.gn

```diff
@@ -37,6 +37,7 @@ chip_test_suite("tests") {
   public_deps = [
     "${chip_root}/src/lib/core",
     "${chip_root}/src/lib/support:testing",
+    "${chip_root}/src/platform",
```

Comment:
This seems like it should be in `src/system/BUILD.gn`.

Reply:
See https://github.com/project-chip/connectedhomeip/pull/25485#discussion_r1127701874